apps/gauzy/src/app/pages/organizations/edit-organization/edit-organization-settings/edit-organization-other-settings/edit-organization-other-settings.component.html

```diff
@@ -468,7 +468,7 @@ <h4 class="settings-section-header">
 					<nb-accordion-item-header>
 						{{ 'ORGANIZATIONS_PAGE.EDIT.BONUS' | translate }}
 					</nb-accordion-item-header>
-					<nb-accordion-item-body>
+					<nb-accordion-item-body #bonusBody [hidden]="bonusBody?.state === 'collapsed' ? true : false">
```

Comment:
```suggestion
					<nb-accordion-item-body #bonusBody [hidden]="bonusBody?.state === 'collapsed'">
```
@coderNadir please simplify condition.